src/lib/components/PromptWithSlider.svelte

```diff
@@ -0,0 +1,145 @@
+<script lang="ts">
+	import type { Section } from '$lib/types';
+
+	export let value: number;
```

Comment:
Interesting that we're using the `section` values for everything except value. Curious if you tried binding to `section.value` instead and if there was bad behavior there?

Reply:
Indeed, I was getting this error

```
ownership_invalid_mutation
src/lib/components/PromptWithSlider.svelte mutated a value owned by src/routes/+page.svelte. This is strongly discouraged. Consider passing values to child components with `bind:`, or use a callback instead
```

I could do an `onchange` handler instead if preferred!

Reply:
although the handler would still need to be passed separately from the parent to avoid this error

Reply:
I think what you have is fine, was just curious! Thanks :)